tests/autopilot/autopilot_test.go

```diff
@@ -245,14 +246,19 @@ var _ = Describe(fmt.Sprintf("{%sRestartAutopilot}", testSuiteName), func() {
 			},
 		}
 
+		// Set driver namespace
+		driverNamespace, err := schedops.GetDriverNamespace()
+		Expect(err).NotTo(HaveOccurred(),
+			fmt.Sprintf("Failed to get driver namespace. Error: [%v]", err))
+
```

Comment:
make it more robust and add GetAutopilotNamespace method. This is confusing as we get the driver namespace and use it for autopilot namespace.

Reply:
I'm not getting AutopilotNamespace here, I'm getting driver namespace.

Reply:
Set this for TODO